src/pages/signup/Signup.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Col, Container, Row, Spinner } from 'reactstrap';
+import { Redirect, RouteProps } from 'react-router-dom';
+
+import AuthHeader from 'components/work/AuthHeader';
+import UserTestimonials from 'components/work/UserTestimonials';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import { AUTHENTICATION_CONTAINER, AUTHENTICATION_CONTENT } from 'styled/sharedClasses/authentication';
+import { VIEWPORT_CENTER_LAYOUT } from 'styled/sharedClasses/layout';
+
+import SignupForm from './SignupForm';
+
+const Signup = (props: RouteProps) => (
+  <FirebaseConsumer>
+    {({ loading, user }: FirebaseUserProps) => {
+      if (loading) {
+        return (
+          <Container className={VIEWPORT_CENTER_LAYOUT}>
+            <Spinner color="primary" style={{ width: '8rem', height: '8rem' }} type="grow" />
```

Comment:
can you hide this behind a "Loading" component so it's consistent

Reply:
👍

Reply:
i'll address this in a separate PR